src/app/clark.routing.ts

```diff
@@ -9,7 +9,7 @@ import { UnauthorizedComponent } from './unauthorized.component';
 const clark_routes: Routes = [
   {
     path: 'auth',
-    loadChildren: () => import('app/auth/auth.module').then(m => m.AuthModule),
+    loadChildren: () => import('app/new-auth/new-auth.module').then(m => m.NewAuthModule),
```

Comment:
Since this is all going to branch off of this branch we can just point auth to the newAuthModule.